src/main/resources/assets/goodgenerator/data/essentia.json

```diff
@@ -412,7 +382,7 @@
       "name": "Terminus",
       "fuelValue": 300000,
       "category": "UNSTABLE",
-      "consumeCeo": 10.0
+      "consumeCeo": 100.0
```

Comment:
No. Just no. This is literally the embodiment of everything wrong with the LEG as it is that just exacerbates the wisp lag issue

Reply:
ibid